jmix-superset/superset/src/main/java/io/jmix/superset/package-info.java

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) Haulmont 2024. All Rights Reserved.
+ * Use is subject to license terms.
+ */
+
+@NonNullApi
+package io.jmix.superset;
+
+import org.springframework.lang.NonNullApi;
```

Comment:
I don't remember what we decided about the add-on status. Should we add the `@Experimental` annotation?

Reply:
Fixed